src/App.tsx

```diff
@@ -1,12 +1,57 @@
 import React from 'react';
 import './App.scss';
+import cn from 'classnames';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+interface State {
+  goods: Good[];
+}
+class App extends React.Component<{}, State> {
+  state: State = {
+    goods: [],
+  };
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  handleGettingGoods = async (optionCallback: () => Promise<Good[]>) => {
+    const goods = await optionCallback();
+
+    this.setState({ goods });
+  };
+
+  render() {
+    return (
+      <>
+        <div className={cn('goods')}>
```

Comment:
not sure if we should use classnames library for one class

Reply:
Я художник, я так бачу

Reply:
![image](https://user-images.githubusercontent.com/60885495/142579863-484a9593-dc6e-4994-926b-a2bae74398e8.png)
```suggestion
        <div className="goods">
```